src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,13 @@
+.catalog {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+
+  &__wraper {
+    max-width: $layout-size;
+    padding: $catalog-gap;
+    display: flex;
+    flex-wrap: wrap;
+    gap: $catalog-gap;
```

Comment:
don't use gap, test wouldn't see it

Reply:
Thanks for good review) I applied all points.